src/modules/dashboard/components/ExpenditurePage/Delay/Delay.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+interface Props {
+  amount?: string;
+  time?: string;
+}
+
+const Delay = ({ amount, time }: Props) => {
+  const getShortTimeSymbol = () => {
+    switch (time) {
+      case 'hours':
+        return 'h';
+      case 'days':
+        return 'day';
+      case 'months':
+        return 'mth';
+      default:
+        return null;
+    }
+  };
+
+  const getTimeSymbol = () => {
+    const val = Number(amount);
+
+    if (val && time === 'hours') {
+      return 'h';
+    }
+    return `${getShortTimeSymbol()}${val > 1 && 's'}`;
+  };
+
+  return (
+    <>
+      {amount}
+      {getTimeSymbol()}
+    </>
+  );
+};
+
+export default Delay;
```

Comment:
After seeing how this needed to be implemented, it is not really necessary to have the shortened version of the time period. We can just have the full time period label for each.